moveit_core/robot_model/src/joint_model.cpp

```diff
@@ -238,5 +238,85 @@ std::ostream& operator<<(std::ostream& out, const VariableBounds& b)
   return out;
 }
 
+
+// Calculating hash for JointModel
+namespace std
+{
+	template<> struct hash<JointModel>
+	{
+		std::size_t operator()(JointModel const& joint) const noexcept
+		{
+			std::size_t h =0;
+      // use Joint name_
```

Comment:
is this intended (i.e. the fact that the comments are now indented)

Reply:
Ah, not sure why github always messes with my indentations. I will need to look into that.